doc/howtos/booting-and-installation.rst

```diff
@@ -127,47 +128,52 @@ Intel Edison
 
 Flashing an Intel Edison requires use of a breakout board and two micro-USB cables:
 
-#. Install the ``dfu-util`` package. (You may also need the ``xfstk`` utility from http://xfstk.sourceforge.net 
+#. Install the ``dfu-util`` package. (You may also need the ``xfstk`` utility from http://xfstk.sourceforge.net
    for recovery cases.)
-#. Plug in a micro-USB cable to the J3 connector on the board (corner next to the FTDI chip)
-#. Flip the DIP switch towards jumper J16
-#. Open :command:`minicom` or other terminal program on your host computer to attach to the serial console
-#. Download the ``flashall`` folder from the Ostro OS download folder for edison (on https://download.ostroproject.org)
-#. Copy the flashall script (``flashall.sh``) from the flashall folder to the Ostro OS image folder
-#. Then in the image folder run:: 
+#. Plug in a micro-USB cable to the J3 connector on the board (corner next to the FTDI chip).
+#. Flip the DIP switch towards jumper J16.
+#. Download the ``ostro-image`` or ``ostro-image-dev`` image from the Ostro OS download folder for
+   Edison (on https://download.ostroproject.org/releases/ostro-os/milestone/).
+#. Extract the image from the archive using the command::
+
+   $ tar xf ostro-image-*-edison-*.toflash.tar.bz2
+
+#. Change directory to the toFlash folder.
+#. Run the command::
+
+   $ sudo ./flashall.sh
 
-    $ sudo ./flashall.sh
+   `NOTE:` If the script is unable to find the image, use the ``-i <imagename>`` option to the flashall script.
```

Comment:
Note: with #399 this *is* the case if user downloads ostro-image-dev and flashes it.